kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TableNameUtils.java

```diff
@@ -42,4 +47,35 @@ public static String destTable(TableId table) {
   public static TableId tableId(TableName name) {
     return TableId.of(name.getProject(), name.getDataset(), name.getTable());
   }
+
+  public static String[] getDataSetAndTableName(BigQuerySinkTaskConfig config, String topic) {
```

Comment:
In the latest 2.6.x - I see this method is present in BigQuerySinkTask class as well. Could you please make sure we do not cause any duplication? Also please pull in the latest 2.6.x branch. Thanks!

Reply:
That's right Aniket. In PR #312  (https://github.com/confluentinc/kafka-connect-bigquery/pull/312/files#diff-ad5086c1123b437f762b169166c133be1b56528bd22eba426ec7dea10110f9b2L191) we are removing it. 
Will pull the latest. Thanks!